service/src/main/kotlin/fi/espoo/evaka/attendance/StaffAttendanceQueries.kt

```diff
@@ -565,6 +566,26 @@ FROM staff_attendance_realtime WHERE employee_id = :employeeId AND departed IS N
         .findOne()
         .orElseGet { null }
 
+fun Database.Read.getLatestDepartureToday(
+    employeeId: EmployeeId,
+    clock: EvakaClock
+): StaffAttendance? =
+    createQuery(
+            """
+SELECT id, employee_id, group_id, arrived, departed, occupancy_coefficient, type
+FROM staff_attendance_realtime 
+WHERE employee_id = :employeeId 
+    AND departed IS NOT NULL
+    AND DATE(departed) = :today
```

Comment:
This probably doesn't work around midnight because `departed` is in UTC and `:today` is in Helsinki timezone.

Reply:
Discussed in tech coffee break: This also doesn't use indexes very well, so it will become slow when an employee has lots of attendances.

Reply:
Agreed to leave the index out